### DISCUSSION TITLE
where is the css var --b2 and --bc coming from?

### DISCUSSION THREAD

**Discussion by lsbyerley** [saadeghi/daisyui#527] [Q&A] [answered]
Doing a search for them in the repo yields no results, just curious what values they are. I've noticed they are used for the background of cards in the daisyui docs

**Comment by saadeghi** [comment, marked as answer]
See this table for details: https://daisyui.com/docs/colors/  
There are 3 shades of base colors that can be used for backgrounds.
- base-100 (--b1)
- base-200 (--b2)
- base-300 (--be)

And `base-content` (--bc) which is used for the content or text.


If b2 and b3 are not specified in a custom theme, they will be darker shades of b1 automatically. And if bc is not defined, it will be an opposite contrast of b1 color.
